### DISCUSSION TITLE
PS Dualshock Rumble

### DISCUSSION THREAD

**Discussion by MaxJumper** [darthcloud/BlueRetro#154] [Q&A] [answered]
I want to thank you for this wonderful project. Thanks
But I have question: The concerns vibration for Playstation 1 and 2 games, there are two motors on the original PS gamepad, with a large and small load. I connected the XBOX ONE gamepad, launched the gamepad tester on the PS2, and only the motor with a small load always works. And when you run a check for a large load and when you run a check for a small load. I didn't find any information on this issue in the roadmap.

**Comment by darthcloud** [comment, marked as answer]
Yes this is normal see #20 . No ETA on when I will fix.
